app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -1,14 +1,12 @@
-function initNotifications() {
-  fetchNotificationsCount();
-  markNotificationsAsRead();
-  initReactions();
-  listenForNotificationsBellClick();
-  initPagination();
-  initLoadMoreButton();
-}
+'use strict';
+
+/* global checkUserLoggedIn */
+/* global instantClick */
+/* global InstantClick */
+/* global sendHapticMessage */
```

Comment:
Forgot one thing: 

these can be all in one line:

`/* global checkUserLoggedIn, instantClick, InstantClick, sendHapticMessage */`